VisualRecognitionV4.playground/Pages/Collections.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,75 @@
+//:## Collections
+
+import Foundation
+import VisualRecognitionV4
+
+let visualRecognition = setupVisualRecognitionV4()
+var createdCollection = "d156c36c-2ac3-499c-a026-fd6e8f8c1687"
+
+//:### Create a collection
+
+visualRecognition.createCollection(
```

Comment:
use these values (from node)

  name: 'my-collection',
  description: 'A description of my collection',